samples/interactive-tutorials/setup/setup-cleanup.js

```diff
@@ -324,6 +324,37 @@ const purgeUserEvents = async (parent, visitorId) => {
   );
 };
 
+const impersonatedAuthClient = async impersonatedPrincipal => {
+  const {GoogleAuth, Impersonated} = require('google-auth-library');
+
+  // Acquire source credentials:
```

Comment:
The `google-auth-library` dependency must be added to `samples/package.json` if it's used here. Also, I would pull the `require` statement to the top of the file with the other requires.